### PR TITLE
0063 notebooks output fi samples

### PR DESCRIPTION
**Pull request by ciurana-numo**
Resolves #64

### CODE REVIEW

File: coronado/__init__.py [hunk 1]
```diff
@@ -277,8 +277,9 @@ def initialize(klass, serviceURL : str, servicePath : str, auth : object):
             An instance of Auth configured to use the the serviceURL within the
             defined scope
         """
+        servicePath = servicePath if servicePath[0] != '/' else servicePath[1:]
+        klass._servicePath = servicePath if servicePath[-1] != '/' else servicePath[:-1]
```
Comment (bruceadams) [inline coronado/__init__.py:281]: `strip` is simpler

```suggestion
        klass._servicePath = servicePath.strip('/')
```

File: coronado/transaction.py [hunk 2]
```diff
@@ -100,7 +101,11 @@ def list(klass: object, paramMap = None, **args) -> list:
         result = [ TripleObject(obj) for obj in json.loads(response.content)['transactions'] ]
         for t in result:
             t.matchingStatus = MatchingStatus(t.matchingStatus)
-            t.merchantCategoryCode = MCC(t.merchantCategoryCode)
+            # TODO: Bug!  Why are there transactions without an MCC?
```
Comment (bruceadams) [inline coronado/transaction.py:104]: I'm not sure we will always get MCC from our upstream.

**Review by bruceadams — APPROVED** [review]

File: coronado/transaction.py (hunk 2 above)
Comment (ciurana-numo) [reply coronado/transaction.py:104]: Aha - thanks.

File: bug-report.ipynb [hunk 3]
```diff
@@ -242,16 +113,71 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "9e2e9c54-a14c-4d3e-b4de-ed7a4337df35",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'eyJraWQiOiJUUTRwU1d0REpnT3drZ3NaZU9COXc1UEx5VkxsWkhiN1g2dCtyRFlWZzBVPSIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiIxcjZxNmkxbXM2Z2pyZTY4cmQ5MjF2b2QxcCIsInRva2VuX3VzZSI6ImFjY2VzcyIsInNjb3BlIjoiYXBpLnRyaXBsZXVwLmNvbVwvcGFydG5lci5jb250ZW50X3Byb3ZpZGVycyIsImF1dGhfdGltZSI6MTY1ODQ1Nzc3NSwiaXNzIjoiaHR0cHM6XC9cL2NvZ25pdG8taWRwLnVzLWVhc3QtMS5hbWF6b25hd3MuY29tXC91cy1lYXN0LTFfck9WZWVLbnhTIiwiZXhwIjoxNjU4NDYxMzc1LCJpYXQiOjE2NTg0NTc3NzUsInZlcnNpb24iOjIsImp0aSI6IjA3NmEzY2MxLWY3MzItNDAwMS04NWY2LTIyNDk4NGNkMGFiZSIsImNsaWVudF9pZCI6IjFyNnE2aTFtczZnanJlNjhyZDkyMXZvZDFwIn0.bKMqV74S4cuLbPjR8ZUhx-Ub46Cb_zHG8zLchYRuJAeiBe6UqBJYV5e5O1PmAfoTckU8fCjgnaZi4JhL0szV3Ssp7r3TKKuLeChsdEP1Q79jj6Mn__PYmuy2BY1XZwgoKhnmrNWj6qe5r-qY2qmeskIXGMuo09wmCSylasFA6Onr2zHul8C4sk1FiosOC2oG6AoGJCTyM_dMXzK4avRHkVjwOn-jclwHh6Q3lEMQksQbCi5rrXiqh7UxmWzlHaBCB2ARPpPvZUu4TA9DruP7_KDFs3NIayyExv4e3K4Rt0ZQVtwEQSu4mMrUXCaJ26hJ0i3qQzYr6GqJZ7A_fx4wiA'"
```
Comment (mfulgo) [inline bug-report.ipynb:149]: Fortunately, this token expired yesterday...

**Pull request merged**